.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "editor.tabSize": 2
```

Comment:
Can you add the .vscode to gitignore list so it's not part of the repo please? It's related to you personal tool and not to project thx :)

Reply:
gotcha